client/my-sites/upgrades/checkout-thank-you/index.jsx

```diff
@@ -199,9 +216,7 @@ const CheckoutThankYou = React.createClass( {
 					primaryPurchase={ primaryPurchase }
 					selectedSite={ this.props.selectedSite } />
 
-				<div className="checkout-thank-you__features-header">
-					{ this.translate( "Get started with your site's new features" ) }
-				</div>
+				{ ! this.shouldHideFeaturesHeading() && <div className={ featuresHeaderClasses }>{ this.translate( 'What now?' ) }</div> }
```

Comment:
I think all the changes introduced in this pull request should be moved into a new subcomponent (or at least in `<CheckoutThankYouHeader />`).